browser/main/lib/ConfigManager.js

```diff
@@ -141,7 +140,7 @@ function get () {
     const theme = consts.THEMES.find(theme => theme.name === config.editor.theme)
 
     if (theme) {
-      editorTheme.setAttribute('href', `../${theme.path}`)
+      editorTheme.setAttribute('href', win ? theme.path : `../${theme.path}`)
```

Comment:
@incredibleweirdo Is this working on Mac? Just wondering because if the other change in `MarkdownPreview` was required then we should change every occurent from `../` to the absolute path.
But if it's working we can also keep it.

Reply:
Yeah, changing themes appears to work fine.